LICENSE

```diff
@@ -0,0 +1,7 @@
+Copyright (C) 2016-2017 Rob Wu rob@robwu.nl
```

Comment:
I added the brackets. I'd rather not edit history just for mentioning the license type (but I did mention it in the second commit message). If you insist on mentioning "MIT license" in the first commit message let me know and I'll simply create a new PR.